CDCgov/prime-reportstream
# Experience/15059/smoke GitHub action

This PR ...

Added github action workflow for frontend smoke tests to be run against live data.

Test Steps:

Run the Start Frontend Smoke Tests workflow.
You should be prompted to select your admin un/pw for such environment.


## Changes


## Checklist

### Testing


## Linked Issues
- Fixes #15059
